editor/inserter/index.js

```diff
@@ -75,7 +75,10 @@ class Inserter extends wp.element.Component {
 					onClick={ this.toggle }
 					className="editor-inserter__toggle"
 					aria-haspopup="true"
-					aria-expanded={ opened ? 'true' : 'false' } />
+					aria-expanded={ opened ? 'true' : 'false' }
```

Comment:
Not specific to the scope of this pull request, but I wonder if React is smart enough to handle the boolean value without the ternary expression.